docs/modules/user/pages/retrieve.adoc

```diff
@@ -1,104 +1,112 @@
 = Search and retrieve
 
-Trustification allows searching SBOM and VEX data using a builtin search engine so that you can find relevant data quickly. Once found,
-you can also retrieve the original SBOM and VEX data.
+Trustification allows you to search for Software Bill of Materials (SBOM) and Vulnerability Exploitability eXchange (VEX) data using a built-in search engine to find relevant data quickly.
+Once found, you can also retrieve the original SBOM and VEX data.
 
-You can use either the UI or the REST API for searching data in both Bombastic and Vexination.
+You can use the web-based user interface (UI), or the REST APIs for searching SBOM and VEX data.
 
-== UI
+== The Spog UI
```

Comment:
"spog" is for Single Pane of Glass. I guess this should be always written as "SPoG". Then again, I am not even sure I am an expert for the German language :D